user_docs/en/userGuide.t2t

```diff
@@ -916,6 +916,9 @@ This option allows the word that is under the cursor to be displayed in non-cont
 This option allows the braille cursor to be turned on and off.
 It applies to the system caret and review cursor, but not to the selection indicator.
 
+==== Blink Cursor ====
+This option allows the braille cursor to blink. If blinking is turned off the braille cursor will just be static.

```

Comment:
1. Please split this onto two lines, one per sentence.
2. I understand what you mean by "static", but I'm not sure many users will. Perhaps we should say "If blinking is turned off, the cursor will be constantly shown." Or we could change "constantly shown" to "always up" (i.e. the cursor dots are always up on the display).